LayoutTests/fast/events/contextmenu-actions-in-selected-text.html

```diff
@@ -8,6 +8,10 @@
     text-align: center;
     display: inline-block;
 }
+
+* {
+    unicode-bidi: normal;
+}
```

Comment:
Same question: why do we need this rule?

Reply:
Without this change, we hit assertion in Editing Code, where we assume `embed` while it should be `isolate or embed`, which I have to investigate and I wanted to do it as separate change rather than merging in it.

Reply:
We should fix the assertion in editing code.

Reply:
I know but do you want me to tackle this in this patch?

Reply:
Yes, we should not introduce new regressions/assertions. Makes it harder to debug websites when they are hit.

Reply:
Removed test changes and fixed assert - locally ran with no issue.